### ISSUE TITLE
PR rebasing

### ISSUE THREAD

**Issue by teverett** [antlr/grammars-v4#2520]
@KvanTTT  @kaby76 

It would be helpful if we could rebase PR's from GH.   We can force dependabot to rebase its PR's but not other PR's.

Could this help?

https://github.com/cirrus-actions/rebase

**Comment by kaby76** [comment]
@teverett I'm really not an expert at GitHub, Git, and rebasing. I always seem to end up in a bad state, likely because I made a commit public. If I have to rebase, I close the PR, and reopen/redo it from scratch with the latest checked in code.

**Comment by teverett** [comment]
@kaby76  me either.  However I see a numbe of PR's that I'd like to get onto the tree and I *think* they'd pass the automated tests if we could rebase them. I'm trying to figure out how to make that happen.

**Comment by studentmain** [comment]
![image](https://user-images.githubusercontent.com/80723249/154423747-626b721f-b7d1-4cc4-b438-863e594b42b5.png)

There is a rebase merge option in the web UI, but I never use it, I almost always do merge locally then push them.

**Comment by KvanTTT** [comment]
Yes, it's possible to select rebase/squash merging right in GitHub. There is nothing complicated about it. I'm using rebasing/squashing in recent times for grammars because merge commit does not provide any useful info.

**Comment by teverett** [comment]
i think what I'm looking for is a way to rebase without merge.  I'd like to run the tests on the rebased PR and then merge.
